cassandra-release/cassandra-check-release.sh

```diff
@@ -0,0 +1,274 @@
+        apache-cassandra-$1/bin/cassandra -R -f" 2>&1 >procfifo &
+
+    PID=$!
+    success=false
+    while read LINE && ! $success ; do
+        if [[ $LINE =~ "Starting listening for CQL clients on" ]] ; then
+            echo "Binary artefact (JDK ${JDK}) OK"
+            kill "$PID"
+            success=true
+        fi
+    done < procfifo
+    rm -f procfifo
+    wait "$PID"
+    if ! $success ; then
+        echo "Binary artefact (JDK ${JDK}) FAILED"
+    fi
+
+    # test deb package startup
+
+    echo
+    rm -f procfifo
+    mkfifo procfifo
+    docker run -i -v `pwd`/debian:/debian openjdk:${JDK}-jdk-slim-buster timeout 2160 /bin/bash -c "
+        ( apt -qq update ;
+          apt -qq install -y python python3 procps ;
+          dpkg --ignore-depends=java7-runtime --ignore-depends=java8-runtime -i debian/*.deb ) 2>&1 >/dev/null ;
+        CASSANDRA_CONF=file:///etc/cassandra/ HEAP_NEWSIZE=500m MAX_HEAP_SIZE=1g cassandra -R -f" 2>&1 >procfifo &
+
+    PID=$!
+    success=false
+    while read LINE && ! $success ; do
+        if [[ $LINE =~ "Starting listening for CQL clients on" ]] ; then
+            echo "Debian package (JDK ${JDK}) OK"
+            kill "$PID"
+            success=true
+        fi
+    done < procfifo
+    rm -f procfifo
+    wait "$PID"
+    if ! $success ; then
+        echo "Debian package (JDK ${JDK}) FAILED"
+    fi
+
+    # test deb repository startup
+
+    idx=`expr index "$1" -`
+    if [ $idx -eq 0 ]
+    then
+        release_short=${1}
+    else
+        release_short=${1:0:$((idx-1))}
+    fi
+    debian_series="$(echo ${release_short} | cut -d '.' -f 1)$(echo ${release_short} | cut -d '.' -f 2)x"
+
+    echo
+    rm -f procfifo
+    mkfifo procfifo
+    docker run -i openjdk:${JDK}-jdk-slim-buster timeout 2160 /bin/bash -c "
+        ( echo 'deb https://dist.apache.org/repos/dist/dev/cassandra/${1}/debian ${debian_series} main' | tee -a /etc/apt/sources.list.d/cassandra.sources.list ;
+          apt -qq update ;
+          apt -qq install -y curl gnupg2 ;
+          apt-key adv --keyserver keyserver.ubuntu.com  --recv-key E91335D77E3E87CB ;
+          curl https://downloads.apache.org/cassandra/KEYS | apt-key add - ;
+          apt update  ;
+          apt-get install -y cassandra ) 2>&1 >/dev/null ;
+        cassandra -R -f" 2>&1 >procfifo &
+
+    PID=$!
+    success=false
+    while read LINE && ! $success ; do
+        if [[ $LINE =~ "Starting listening for CQL clients on" ]] ; then
+            echo "Debian repository (JDK ${JDK}) OK"
+            kill "$PID"
+            success=true
+        fi
+    done < procfifo
+    rm -f procfifo
+    wait "$PID"
+    if ! $success ; then
+        echo "Debian repository (JDK ${JDK}) FAILED"
+    fi
+
+    if [ "$JDK" == "8" ] ; then
+        JDK_RH="java-1.8.0-openjdk"
+    elif [ "$JDK" == "11" ] ; then
+        JDK_RH="java-11-openjdk-devel"
+    fi
+
+    RH_DISTS="almalinux"
+    if ! [[ $1 =~ [23]\. ]] ; then
```

Comment:
It should be without `!`, if we test Cassandra 2 / 3, we want to test it against Centos 7, no? We would drop support of Centos 7 for newer releases.